heareval/predictions/task_predictions.py

```diff
@@ -413,6 +424,17 @@ def __init__(
         self.epoch_best_postprocessing: Dict[int, Tuple[Tuple[str, Any], ...]] = {}
         self.postprocessing_grid = postprocessing_grid
 
+    def epoch_best_postprocessing_or_default(
+        self, epoch: int
+    ) -> Tuple[Tuple[str, Any], ...]:
+        if self.use_scoring_for_early_stopping:
+            return self.epoch_best_postprocessing[epoch]
+        else:
+            postprocessing_confs = list(ParameterGrid(self.postprocessing_grid))
```

Comment:
where do we define the postprocessing grids for secret tasks?

Reply:
It's the default unless we override it using config `evaluation_params: event_postprocessing_grid: {}`